src/site/content/en/blog/cors-rfc1918-feedback/index.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Feedback wanted: CORS for private networks (RFC1918)
+subhead: |
+  Mitigate the risks associated with unintentional exposure of devices and servers on a client’s internal network to the web at large.
+description: |
+  Unintentional exposure of devices and servers on a client’s internal network to the web at large makes them 
+  vulnerable to malicious attacks. CORS-RFC1918 is a proposal to block requests from public networks by default 
+  on the browser and require internal devices to opt-in to such requests.
+authors:
+  - agektmr
+date: 2020-11-09
+hero: hero.jpg
+alt: The hands of a person playing the Chrome dino game on a game console.
```

Comment:
When I preview the page I see a close-up pic of ethernet ports on what seems to be a router. This seems to be confirmed by the hero image credit at the bottom of the page.

Reply:
Good catch! Fixed.